pkgs/development/tools/mold/default.nix

```diff
@@ -12,13 +12,13 @@
 
 stdenv.mkDerivation rec {
   pname = "mold";
-  version = "1.6.0";
+  version = "1.7.0";
 
   src = fetchFromGitHub {
     owner = "rui314";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-IXXyZZw1Tp/s9YkPR5Y+A6LpvaRo+XfA8UJBtt5Bjmg=";
+    hash = "sha256-IXXyZZw1Tp/s9YkPR5Y+A6LpvaRo+XfA8UJBtt5Bjmh=";
```

Comment:
That doesn't look right, the hash should be completely different

Reply:
This hash is definitively incorrect, it should be `hSDGVS3Q0LJ7HZPz6/kSRK+9XejETra5FloeEMrkdLg=`.

Reply:
Oops, yes, you are right. Sorry, I pushed the wrong commit.